source/docs/software/python/deploying-robot-code.rst

```diff
@@ -0,0 +1,75 @@
+
+Deploying Robot Code
+====================
+
+.. note:: Before deploying the code to your robot, you must start by :ref:`installing RobotPy on your roboRIO <docs/software/python/installing-robotpy-on-roborio:RobotPy installation on roboRIO>`
+
+To deploy code to your robot, you can just run the following command and it will upload the code and start it immediately.
+
+.. tabs::
+   .. group-tab:: Windows
+
+    .. code-block:: sh
+
+        py -3 robot.py deploy
+
+   .. group-tab:: Linux/macOS
+
+    .. code-block:: sh
+
+        python3 robot.py deploy
+
+You can watch your robot code's output (and see any problems) by using the netconsole program (you can either use NI's tool, or `pynetconsole <https://github.com/robotpy/pynetconsole>`__. You can use netconsole and the normal FRC tools to interact with the running robot code.
+
+If you're having problems deploying code to the robot, you can `join the Discord channel <https://discord.gg/Vj3DWnBr>`__ and ask for help!
```

Comment:
I don't think we have an officially endorsed discord channel

Reply:
Nor has RobotPy either.

Reply:
Sorry about that, it was a placeholder I forgot about.

Reply:
I am looking for the best place to send newcomers. My guess is that the appropriated official resource is the [FIRST Inspire Programming Forum](https://forums.firstinspires.org/forum/general-discussions/first-programs/first-robotics-competition/competition-discussion/programming-aa), but do you think it would be possible to have a Python section?